test-apps/display-test-app/src/common/DtaConfiguration.ts

```diff
@@ -108,6 +109,9 @@ export const getConfig = (): DtaConfiguration => {
   if (undefined !== process.env.IMJS_DISABLE_BREP_CACHE)
     configuration.disableBRepCache = true;
 
+  if (undefined !== process.env.IMJS_DISABLE_UNIFORM_ERRORS)
```

Comment:
Update README to document this new env var.

Reply:
afaik webpack does not compile out `console.log` and that works, and i think thats what david had said.
i think he was saying the `Logger` from `core-bently` doesnt log in prod which i need to look into....

Reply:
@aruniverse repro steps:
Apply following diff:
```
diff --git a/core/frontend/src/render/webgl/UniformHandle.ts b/core/frontend/src/render/webgl/UniformHandle.ts
index 33d668c070..cf08076063 100644
--- a/core/frontend/src/render/webgl/UniformHandle.ts
+++ b/core/frontend/src/render/webgl/UniformHandle.ts
@@ -44,6 +44,7 @@ export class UniformHandle {
     if (undefined !== program.glProgram) {
       location = System.instance.context.getUniformLocation(program.glProgram, name);
     }
+    Logger.logError(FrontendLoggerCategory.Render, "hello");
     if (null === location) {
       const errMsg = `uniform ${name} not found in ${program.description}.`;
       if (System.instance.options.errorOnMissingUniform) {
diff --git a/test-apps/display-test-app/src/frontend/DisplayTestApp.ts b/test-apps/display-test-app/src/frontend/DisplayTestApp.ts
index 246fd1ff9d..92cbf36948 100644
--- a/test-apps/display-test-app/src/frontend/DisplayTestApp.ts
+++ b/test-apps/display-test-app/src/frontend/DisplayTestApp.ts
@@ -2,7 +2,7 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { ProcessDetector } from "@itwin/core-bentley";
+import { Logger, LogLevel, ProcessDetector } from "@itwin/core-bentley";
 import { CloudStorageContainerUrl, CloudStorageTileCache, RpcConfiguration, TileContentIdentifier } from "@itwin/core-common";
 import { IModelApp, IModelConnection, RenderDiagnostics, RenderSystem, TileAdmin } from "@itwin/core-frontend";
 import { WebGLExtensionName } from "@itwin/webgl-compatibility";
@@ -184,9 +184,9 @@ const dtaFrontendMain = async () => {
 
     await uiReady; // Now wait for the HTML UI to finish loading.
     await initView(iModel);
-    // Logger.initializeToConsole();
-    // Logger.setLevelDefault(LogLevel.Warning);
-    // Logger.setLevel("core-frontend.Render", LogLevel.Error);
+    Logger.initializeToConsole();
+    Logger.setLevelDefault(LogLevel.Warning);
+    Logger.setLevel("core-frontend.Render", LogLevel.Error);
 
     if (configuration.startupMacro)
       await IModelApp.tools.parseAndRun(`dta macro ${configuration.startupMacro}`);
```

rush clean
rush build --to display-test-app
npm run start
Open a view.
Observe many "hello"s in browser console.

rush clean
rush --build:ci --to display-test-app
npm run start
Open a view
Observe no "hello"s in browser console.

NODE_ENV is not set.

Reply:
In the latter case, we call into Logger.logError, where Logger._logError is undefined.

Reply:
`Logger.initialize` is invoked at startup and I see it setting _logError to a function from `Logger.initializeToConsole`.
Static variable problems?

Reply:
Yes, apparently Logger does not log in prod, which needs fixing but I will continue with this PR.